src/shared/components/mutationTable/column/ChromosomeColumnFormatter.spec.js

```diff
@@ -12,7 +12,7 @@ describe('ChromosomeColumnFormatter functions', () => {
     let e = "Y";
     let f = "chrY";
 
-    it('chromosome integer values are properly extracted', () => {
+    it('extractSortValue function', () => {
```

Comment:
These tests are supposed to read like sentences, like `it('properly extracts chromosome integer values', () => {`